src/test/java/com/cloudbees/hudson/plugins/folder/FolderTest.java

```diff
@@ -278,6 +286,32 @@ private void copyFromGUI(Folder f, JenkinsRule.WebClient wc, String fromName, St
             }
         });
     }
+    
+    @Issue("JENKINS-32359")
+    @Test public void shouldProperlyPersistFolderPropertiesOnMultipleReloads() throws Exception {
+        Folder folder = r.jenkins.createProject(Folder.class, "myFolder");
+        r.jenkins.setAuthorizationStrategy(new ProjectMatrixAuthorizationStrategy());
+        
+        // We add a stub property to generate the persisted list
+        // After that we save and reload the config in order to drop PersistedListOwner according to the JENKINS-32359 scenario
+        folder.addProperty(new FolderCredentialsProvider.FolderCredentialsProperty(new DomainCredentials[0]));
```

Comment:
what is this first part doing that is important to the test that the second part is not doing?

i.e. why 2 properties and 2 reloads - yet you are only asserting on one property being persisted?